cmd/toitpkg/main.go

```diff
@@ -107,89 +78,21 @@ func main() {
 		return nil
 	}
 
-	pkgCmd, err := commands.Pkg(runWrapper, track, &viperConfigStore{}, nil)
+	configStore := store.NewViper(cacheDir, sdkVersion, noAutosync, noDefaultRegistry)
```

Comment:
This doesn't work as wanted.
All arguments can be changed by flags to the executable.
However, the flags are only evaluated later, so we always get the default values here.